['text':' Owner(s): ["oncall: mobile"]','line_number':1,'multiline':False]
['text':' opt arg in script-to-script invocation','line_number':157,'multiline':False]
['text':' opt arg in Python-to-script invocation','line_number':166,'multiline':False]
['text':' change ref only','line_number':184,'multiline':False]
['text':' now both match again','line_number':192,'multiline':False]
['text':' In C++ code, the type of exception thrown is torch::jit::JITException','line_number':328,'multiline':False]
['text':' which does not extend c10::Error, and hence it isn't possible to add','line_number':329,'multiline':False]
['text':' additional context to the exception message and preserve the correct','line_number':330,'multiline':False]
['text':'  C++ stack trace for symbolication. i.e. it isn't possible to add','line_number':331,'multiline':False]
['text':' the debug handle string to show where in the Python code the exception','line_number':332,'multiline':False]
['text':' occured w/o first changing','line_number':333,'multiline':False]
['text':' torch::jit::JITException to extend c10::Error.','line_number':334,'multiline':False]
['text':' In C++ code, the type of exception thrown is torch::jit::JITException','line_number':407,'multiline':False]
['text':' which does not extend c10::Error, and hence it isn't possible to add','line_number':408,'multiline':False]
['text':' additional context to the exception message and preserve the correct','line_number':409,'multiline':False]
['text':'  C++ stack trace for symbolication. i.e. it isn't possible to add','line_number':410,'multiline':False]
['text':' the debug handle string to show where in the Python code the exception','line_number':411,'multiline':False]
['text':' occured w/o first changing','line_number':412,'multiline':False]
['text':' torch::jit::JITException to extend c10::Error.','line_number':413,'multiline':False]
['text':' From the example in Static Quantization section of https://pytorch.org/docs/stable/quantization.html','line_number':482,'multiline':False]
